### DISCUSSION TITLE
Encrypting wasm in browser

### DISCUSSION THREAD

**Discussion by dhanrajmahurkar** [wasmerio/wasmer#4571] [Ideas]
I would like to run my wasm file in the browser while also protect it from being reverse engineered. Is there any way to do that? I can inspect network requests from chrome browser and get the wasm file. What steps can be taken to mitigate this?

**Comment by syrusakbary** [comment]
This is an awesome idea. Would you like to schedule some time to chat so we can learn more about your use case?

Feel free to select a time that works for you here: https://cal.com/syrus
